src/ngraph/op/util/attr_types.hpp

```diff
@@ -298,4 +326,52 @@ namespace ngraph
         static constexpr DiscreteTypeInfo type_info{"AttributeAdapter<op::AutoBroadcastSpec>", 0};
         const DiscreteTypeInfo& get_type_info() const override { return type_info; }
     };
+
+    namespace op
+    {
+        /// \brief Implicit broadcast specification
+        struct NGRAPH_API BroadcastModeSpec
```

Comment:
@diyessi @lazarevevgeny  Can we break `PDPD` support for `Broadcast:v3`? (it will significantly simplify the code but break backward compatibility)

Reply:
We should not do it. I understand that the code complexity may increase, but we must keep backward compatibility